### ISSUE TITLE
Public page for projects

### ISSUE THREAD

**Issue by combojs** [mozilla/thimble.mozilla.org#2601]
I was wondering if there was a way to share a link to my list of projects. If there isn't a way, is this a feature that can be discussed for future versions?

**Comment by humphd** [comment]
This is not a way to do this, no.  It could be built, though, especially since we already have code that basically does this for a user.  Exposing it might not be what some users want, though.  This needs some thought.

**Comment by combojs** [comment]
Good point. Maybe have an option to showcase your published projects or not.

**Comment by gideonthomas** [comment]
it might be a bit of work, but we can possibly do this. We can add a view to thimble under a url like `thimble.mozilla.org/<myusername>/projects/published` that queries the publish server (we'll need to add a route to it to handle this) to return the published projects for a user by their username.

**Comment by combojs** [comment]
That would be great. I know some sandboxes like Codepen offer this.

**Comment by c0dzilla** [comment]
I would love to work on this! May I take up this issue?

**Comment by combojs** [comment]
Has anyone looked at this feature request. Is it possible to do in Thimble?
